Mono.Debugging/Mono.Debugging.Client/ProcessInfo.cs

```diff
@@ -64,14 +64,14 @@ internal void Attach (DebuggerSession session)
 
 		public ProcessInfo (string name, string description)
 		{
-			this.name = name;
-			this.description = description;
+			this.name = name ?? string.Empty;
+			this.description = description ?? string.Empty;
 		}
 
 		public ProcessInfo (long id, string name)
 		{
 			this.id = id;
-			this.name = name;
+			this.name = name ?? string.Empty;
```

Comment:
this leaves the `description` field (and the associated property) to return `null`, which is what you seem to want to avoid, right ?

Reply:
You're right. The `description` field wasn't the one causing issues (it was just `name`), so I'll revert the change to it